.github/.jira_sync_config.yaml

```diff
@@ -0,0 +1,18 @@
+# From https://github.com/canonical/gh-jira-sync-bot#client-side-configuration
+settings:
+  jira_project_key: "IAM"
+  status_mapping:
+    opened: Untriaged
+    closed: done
+  components:
+    - Oathkeeper
+  labels:
+    - bug
+    - enhancement
+  add_gh_comment: true
+  sync_description: true
+  sync_comments: true
+  epic_key: "IAM-471"
```

Comment:
Shouldn't it be the epic that is linked with the repo, so https://warthogs.atlassian.net/browse/IAM-30 in this case? Or will IAM-471 epic be responsible for collecting bugs from all repos?